src/components/ble/SimpleWeatherService.h

```diff
@@ -112,6 +112,10 @@ namespace Pinetime {
         return celsius * 9 / 5 + 3200;
       }
 
+      static int16_t TempToUnits(int16_t temp) {
```

Comment:
I think this name is not quite representative, since it doesn't include units. Maybe something like "RenderTemp" or "DisplayTemp" or "RoundTemp"? Not entirely sure, naming is hard :/

Other than that, I really like what you're doing centralizing this function.

Reply:
What is the meaning of the input and output?

Reply:
The input is a temperature in units multiplied by 100, and the output is temperature in the actual unit. I wasn't sure of the naming either, it's a surprisingly difficult operation to give a name to.

Reply:
`CentiToUnit`? As in centicelsius to celsius or centifahrenheit to fahrenheit.

Reply:
This method could simply be called `Convert()` if we used strong types for arguments. Ex: 
```c++
static DisplayTemperature Convert(const SimpleWeatherService::RawTemperature temperature)
```

Using stronger types for
 - temperature coming from the weather service (in "centi celsius degrees")
 - temperature in °C
 - temperature in °F

Would probably help with code readability.